packages/venia-ui/lib/components/Modal/index.js

```diff
@@ -1 +1,2 @@
-export { default as Modal } from './modal';
+// To keep backwards compatibility, Modal is an alias for Portal.
```

Comment:
Good work. Perhaps leave a deprecation notice or at least a TODO. I think Zetlen has some examples of deprecation notices in the codebase.